crates/sui-config/src/transaction_deny_config.rs

```diff
@@ -71,6 +75,17 @@ pub struct TransactionDenyConfig {
     /// A list of disabled OAuth providers for zkLogin
     #[serde(default)]
     zklogin_disabled_providers: HashSet<String>,
+
+    /// Dynamic transaction checks to run on transactions.
+    /// Program is loaded at deserialization time to ensure that any syntactic issues are caught
+    /// immediately.
```

Comment:
For my own edification, can you show me where this is called when starting the node?

Reply:
Sure thing!

This is a field/section of a the `validator.yaml`. 

So in particular, this `TransactionDenyConfig` is a field of the `NodeConfig` config: https://github.com/MystenLabs/sui/blob/main/crates/sui-config/src/node.rs#L139

And this gets deserialized/loaded as one of the first things when starting up a node here: https://github.com/MystenLabs/sui/blob/main/crates/sui-node/src/main.rs#L55

Where the `load` function is a trait method defined here: https://github.com/MystenLabs/sui/blob/main/crates/sui-config/src/lib.rs#L100 and implemented for the `NodeConfig` here: https://github.com/MystenLabs/sui/blob/main/crates/sui-config/src/node.rs#L668